src/clib/lib/enkf/field_trans.cpp

```diff
@@ -239,16 +196,6 @@ field_trans_table_type *field_trans_table_alloc() {
                           "This function will calculate y = exp(x) - 0.000001",
                           field_trans_exp0);
 
-    // Rubakumar specials:
-    field_trans_table_add(table, "NORMALIZE_PERMX", "...", normalize_permx);
-    field_trans_table_add(table, "DENORMALIZE_PERMX", "...", denormalize_permx);
```

Comment:
Do we have any way of knowing whether the "Rubakumar specials" are actually used?

Reply:
That would have to be looking at the azure logs i guess? Can we search the configurations used/printed there to check that?

Reply:
I find no hits on azure logs when searching for 'NORMALIZE'. This should catch all Rubakumar specials if they were executed on azure. I'm not sure how far back the logs do go though.
I've verified the search pattern by looking for other valid functions, namely  POW, LOG etc. They are used quite extensively.

As a side note, INPUT_TRANSFORM is present in the code, but not found on azure, nor mentioned in the FMU docs. Not to be mixed with INIT_TRANSFORM which is in use.

Reply:
Frode wrote on slack: Yes. I did not implement those transformation for the same reasons mentioned above

Reply:
I think we can say that this is generally not used then, and in any case we already had a plan to remove it.